unleash/Unleash.swift

```diff
@@ -78,12 +75,14 @@ public class Unleash {
     self.strategies = strategies
     
     register(body: clientRegistration)
-    .then { self.toggleRepository.get(url: URL(string: url)!) }
-    .done { _ in self.delegate?.unleashDidFetchToggles(self) }
-    .catch { error in
-      log("error \(error)")
-      self.delegate?.unleash(self, didFailWithError: error)
+    .then { self.fetchToggles() }
+    .then { _ -> Promise<Void> in
+      if let interval = self.refreshInterval {
+        self.pollFeatures(TimeInterval(interval))
+      }
+      return .value(())
```

Comment:
Question about Swift -- what? `.value(())`?

Reply:
Haha, great question @johnny-1111one1 . `.value()` is a class method in PromiseKit that instantiates and returns a resolved promise with the passed in value. In the case above, `()` is Swift short-hand for `Void`.